internal/k8s/crdcontroller.go

```diff
@@ -964,7 +964,12 @@ func checkRefOnController(key, refKey, refValue string) error {
 // update internal CRD caches, and push relevant ingresses to ingestion
 func validateHTTPRuleObj(key string, httprule *akov1alpha1.HTTPRule) error {
 	refData := make(map[string]string)
+	var pkiDestCASet bool
 	for _, path := range httprule.Spec.Paths {
+		if path.TLS.PKIProfile != "" && path.TLS.DestinationCA != "" {
+			pkiDestCASet = true
```

Comment:
shall we return the function from here itself?

Reply:
addressed